packages/controllers/src/subject-metadata/SubjectMetadataController.ts

```diff
@@ -0,0 +1,261 @@
+import {
+  BaseControllerV2 as BaseController,
+  RestrictedControllerMessenger,
+} from '@metamask/controllers';
+import type { Patch } from 'immer';
+import { Json } from 'json-rpc-engine';
+
+import type {
+  GenericPermissionController,
+  PermissionSubjectMetadata,
+  HasPermissions,
+} from '../permissions';
+
+const controllerName = 'SubjectMetadataController';
+
+type SubjectOrigin = string;
+
+export type SubjectMetadata = PermissionSubjectMetadata & {
```

Comment:
The old subject metadata included a `lastUpdated` property. Was that removed intentionally? I'm not sure what that was intended for to be honest. It appears to be unused at the moment.

Reply:
I had forgotten about this property, but I think we can get rid of it.

IIRC it was intended for the rosy future when we can fetch metadata on demand, and use `lastUpdated` to determine when to do so. However, since fetching metadata is cheap, we can probably just get rid of it?